packages/overhead-metrics/src/scenarios.ts

```diff
@@ -53,3 +53,28 @@ export class JankTestScenario implements Scenario {
     await new Promise(resolve => setTimeout(resolve, 12000));
   }
 }
+
+export class BookingAppScenario implements Scenario {
+  public constructor(private _indexFile: string, private _count: number) {}
+
+  /**
+   *
+   */
+  public async run(_: playwright.Browser, page: playwright.Page): Promise<void> {
+    let url = path.resolve(`./test-apps/booking-app/${this._indexFile}`);
+    assert(fs.existsSync(url));
+    url = `file:///${url.replace(/\\/g, '/')}?count=${this._count}`;
+    console.log('Navigating to ', url);
+    await page.goto(url, { waitUntil: 'load', timeout: 60000 });
+
+    // Click "Update"
+    await page.click('#search button');
+
+    for (let i = 1; i < 10; i++) {
+      await page.click(`.result:nth-child(${i}) [data-select]`);
+    }
+
+    // Wait for flushing, which we set to 2000ms
+    await new Promise(resolve => setTimeout(resolve, 2000));
```

Comment:
Should we wait a little longer than 2000ms considering the timing inconsistencies we observed in #7266?

Reply:
Hmm, true I guess! I'll extend it to 4000ms to be safe.